### PR TITLE
Features/color pallete

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Color pallet feature added to new note

### CODE REVIEW

**Review by poojakelkar — APPROVED** [review]
looks good to me

File: src/component/color-pallet/ColorPallet.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  {
+    id: uuid(),
+    color: '#cfe1b9'  
+  },
+  {
+    id: uuid(),
+    color: '#ccff90'  
+  },
+  {
+    id: uuid(),
```
Comment (ark2002) [inline src/component/color-pallet/ColorPallet.jsx:23]: this was really a smart approach of doing the color thing

File: src/component/color-pallet/color-pallete.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  height: 30px;
+  position: relative;
+}
+.color-btn{
+  cursor: pointer;
+  width: 30px;
+  height: 30px;
+  border: none;
+  border-radius: 50%;
+}
```
Comment (ark2002) [inline src/component/color-pallet/color-pallete.css:27]: maybe use rems here

File: src/component/color-pallet/ColorPallet.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    color: '#cfe1b9'  
+  },
+  {
+    id: uuid(),
+    color: '#cfe1b9'  
+  },
+  {
+    id: uuid(),
+    color: '#ccff90'
```
Comment (ark2002) [inline src/component/color-pallet/ColorPallet.jsx:20]: this was really a smart approach of implementing the colors

File: src/component/text-card/NoteForm.jsx [hunk 4]
```diff
@@ -100,17 +94,21 @@ export const NoteForm = (
...
                 <LabelOutlinedIcon className="mx-2 icon" />
               </div>
-              <div className="d-flex items-center light-text">
+              {/* <div className="d-flex items-center light-text">
                 <DeleteOutlineOutlinedIcon className="mx-2 icon" />
               </div>
               <div className="d-flex items-cente light-text">
                 <ArchiveOutlinedIcon className="mx-2 icon" />
-              </div>
+              </div> */}
```
Comment (ark2002) [inline src/component/text-card/NoteForm.jsx:111]: remove commented code before commits

**Review by ark2002 — APPROVED** [review]
rest looks great

**Pull request merged**